diag_manager/fms_diag_axis_object.F90

```diff
@@ -237,6 +243,7 @@ subroutine register_diag_axis_obj(this, axis_name, axis_data, units, cart_name,
 
     this%nsubaxis = 0
     this%num_attributes = 0
+    this%structured_ids = diag_null
```

Comment:
Can this be allocatable instead of used for every axis?  Or should we make a new type that extends the fmsDiagFullAxis_type

Reply:
Changed this to allocatables